code/datums/emotes.dm

```diff
@@ -73,10 +73,10 @@
 
 /datum/emote/proc/select_message_type(mob/user)
 	. = message
-	if(!muzzle_ignore && user.is_muzzled() && emote_type == EMOTE_AUDIBLE)
-		return "makes a [pick("strong ", "weak ", "")]noise."
 	if(user.mind && user.mind.miming && message_mime)
 		. = message_mime
+	if(!user.can_speak() && !message_mime && emote_type == EMOTE_SPEAK || user.is_muzzled() && emote_type == EMOTE_SPEAK)
```

Comment:
This for whatever reason checks for `message_mime` being absent, while the proper check for that is two lines above this one.

Please remove that part of this check or explain.